src/pages/CreateUserPage.jsx

```diff
@@ -387,6 +387,8 @@ function CreateUserPage(props) {
                                         id="name"
                                         className="form-control rounded-4 bg-light-pastel-blue color-grey fw-medium  fs-5 mb-3 border-0"
                                         onChange={(e) => setNewUser({ ...newUser, name: e.target.value })}
+                                        minLength="3"
+                                        required
```

Comment:
```suggestion
                                        minLength="3"
                                        maxLength="255"
                                        required
```
- Nome também possui limite superior na API;